src/XamlStyler.dotUltimate/XamlStyler.dotUltimate.sln

```diff
@@ -0,0 +1,48 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Why is nuget.exe included?

Reply:
Removed, was needed for the R# plugin but no for Rider-only.